style.css

```diff
@@ -1,29 +1,62 @@
 * {
   text-transform: capitalize;
-  overflow-x: hidden;
   padding: 0;
   margin: 0;
   font-family: "Poppins", sans-serif;
+  overflow-x: hidden;
+}
+
+html {
+  overflow-x: hidden;
 }
 
 body {
   background-color: #e5e5e5;
   font-family: "Poppins", sans-serif;
   padding: 0;
   margin: 0;
+  overflow-x: hidden;
 }
 
 /* ---------------------------- Phone styles -------------------------------- */
 
-.nav {
-  padding: 20px 15px;
+.header-container {
+  padding: 10% 5%;
```

Comment:
- [ ] [OPTIONAL] Kindly remove this padding on the desktop version to make your design more consistent with the template. Currently, the navbar is looking a bit huge. 😎️

![Screenshot from 2023-01-31 11-44-23](https://user-images.githubusercontent.com/17243325/215711892-02c57389-80f5-41a0-a2e6-3063ff3f87e0.png)